### PR TITLE
[BLOG] Add whats-new-in-sympy-14

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-blog-name-quansight.vercel.app/blog/blog-name).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/whats-new-in-sympy-14.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+title: "What's New in SymPy 1.4"
+published: May 5, 2019
+author: aaron-meurer
+description: 'SymPy 1.4 was released on April 9, 2019. In this post, I'd like to go over some of the highlights for this release.'
```
Comment (MarsBarLee) [inline apps/labs/posts/whats-new-in-sympy-14.md:5]:
```suggestion
description: "SymPy 1.4 was released on April 9, 2019. In this post, I'd like to go over some of the highlights for this release."
```

**Comment by MarsBarLee** [comment]
@gabalafou The Vercel build error shows a prerendering error and ReferenceError:
```
Error occurred prerendering page "/blog/whats-new-in-sympy-14". Read more: https://nextjs.org/docs/messages/prerender-error
ReferenceError: n is not defined
at _createMdxContent (eval at <anonymous> (file:///vercel/path0/node_modules/next-mdx-remote/dist/index.js:55:35), <anonymous>:815:47)
at MDXContent (eval at <anonymous> (file:///vercel/path0/node_modules/next-mdx-remote/dist/index.js:55:35), <anonymous>:974:9)
...
```
([Link to Vercel build error](https://vercel.com/quansight/labs/51f8mGWcr6E5DWzcBpWWYQTx2ijy#:~:text=Error%20occurred%20prerendering%20page%20%22/blog/whats%2Dnew%2Din%2Dsympy%2D14%22.%20Read%20more%3A%20https%3A//nextjs.org/docs/messages/prerender%2Derror))

I think this error is related to the math used in the blog post. Maybe due to the `<math>` tag or `pycon` in the code blocks?

**Comment by gabalafou** [comment]
Hey @MarsBarLee, I pushed a commit to this branch that I think fixes the build issues. If it builds this time, can you double check the post and if it looks good on your end, I can go ahead and approve and squash-merge this PR into `develop`.

File: apps/labs/posts/whats-new-in-sympy-14.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  src: /posts/whats-new-in-sympy-14/blog_feature_org.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/whats-new-in-sympy-14/blog_hero_org.svg
+  imageAlt: 'An illustration of a white hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+As of November, 2018, I have been working at
+[Quansight](https://www.quansight.com/), under the heading of [Quansight
+Labs](https://www.quansight.com/labs). Quansight Labs is a public-benefit
```
Comment (noatamir) [inline apps/labs/posts/whats-new-in-sympy-14.md:17]:
```suggestion
Labs](https://labs.quansight.org). Quansight Labs is a public-benefit
```

Comment (noatamir) [reply apps/labs/posts/whats-new-in-sympy-14.md:17]: updated to our current homepage

**Review by noatamir — APPROVED** [review]
LGTM

**Comment by gabalafou** [comment]
@noatamir since it looks good to you, I'm going to go ahead and merge this is in.

**Pull request merged**